src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2419,7 +2419,6 @@ internal void Run(bool inputLoopIsNested)
                 Dbg.Assert(ui != null, "Host.UI should return an instance.");
```

Comment:
@vexx32, your last commit had 40 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Online help tests for PowerShell Cmdlets.Error occurred in Describe block
```powershell
Cannot validate argument on parameter 'Name'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
At line:16 char:34
```
PowerShellGet - Module tests (Admin).Error occurred in Describe block
```powershell
The variable '$lastError' cannot be retrieved because it has not been set.
At line:16 char:34
```
PowerShellGet - Script tests (Admin).Should install a script correctly to the required location with AllUsers scope
```powershell
The variable '$lastError' cannot be retrieved because it has not been set.
at <ScriptBlock>, : line 16
16:             $foundSuggestion = @(Get-Command ($ExecutionContext.SessionState.Path.Combine(".", $escapedCommand)) -ErrorAction Ignore).Count -gt 0

```
ConsoleHost unit tests.ShellInterop.Verify Validate Dollar Error Populated should throw exception
```powershell
Expected an exception, with FullyQualifiedErrorId 'CommandNotFoundException' to be thrown, but the FullyQualifiedErrorId was 'ParameterArgumentValidationError,Microsoft.PowerShell.Commands.GetCommandCommand'. from /Users/runner/runners/2.164.7/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1:108 char:22
    + ?        $a | & $powershell -noprofile -command { wgwg-wrwrhqwrhrh35h3h ?
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.164.7/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 109
109:             } | Should -Throw -ErrorId "CommandNotFoundException" -PassThru
```
ConsoleHost unit tests.Pipe to/from powershell.errors are in text if error is redirected, encoded command, non-interactive, and outputformat specified
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /Users/runner/runners/2.164.7/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 416
416:             $out | Should -Not -BeNullOrEmpty
```